cli/src/defaults.ts

```diff
@@ -0,0 +1,13 @@
+import { getDatabaseURL } from '@xata.io/client';
```

Comment:
[really-minor] Not sold on the name ``defaults.ts``, but no better suggestions either.